src/views/TravelRequest/index.js

```diff
@@ -160,19 +204,19 @@ const TravelRequest = () => {
               variant="contained"
               className={classes.editBtn}
               onClick={handleEdit}
-              disabled={requestData.requestStatus !== 'PENDING'}
+              // disabled={requestData.requestStatus !== "PENDING"}
             >
-              {t('Edit')}
+              {t("Edit")}
             </Button>
             <Button
               size="small"
               data-testid="submit"
               variant="contained"
               className={classes.submitBtn}
               onClick={submitrequest}
-              disabled={requestData.requestStatus !== 'PENDING'}
```

Comment:
If you remove this line, when a request has status of `APPROVED` or `REJECTED` you will still see those buttons of actioning (like edit and submit) which is not good implementation because when decision was made already those buttons should be disabled since user will have no any other action s/he can perform